### PR TITLE
Change README to link to external docs for native apps

### PR DESCRIPTION
**Pull request by mraible**
Depends on https://github.com/jhipster/jhipster.github.io/pull/1204.

### CODE REVIEW

**Comment by mraible** [comment]
Please review @mshima.

File: generators/ionic/resources/base/README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@
...
 2. [Run on iOS](#run-on-ios)
 3. [Run on Android](#run-on-android)
-4. [Use Okta for Authentication](#use-okta-for-authentication)
-5. [Use Auth0 for Authentication](#use-auth0-for-authentication)
+4. [Use OpenID Connect for Authentication](#use-openid-connect-for-authentication)
+    - Configure [Okta](#use-okta)
+    - Configure [Auth0](#use-auth0)
 6. [Services](#services)
 8. [i18n](#i18n) (adding languages)
 9. [Testing](#testing)
```
Comment (mshima) [inline generators/ionic/resources/base/README.md:20]:
```suggestion
5. [Services](#services)
6. [i18n](#i18n) (adding languages)
7. [Testing](#testing)
```

**Review by mshima — APPROVED** [review]
Looks good, may need to update the links if the titles changes at the jhipster site.

File: generators/ionic/resources/base/README.md [hunk 2]
```diff
@@ -145,172 +146,21 @@ If you see `java.io.IOException: Cleartext HTTP traffic to localhost not permitt
...
 
 ```bash
 docker-compose -f src/main/docker/keycloak up -d
 ```
 
-See [JHipster's security docs](https://www.jhipster.tech/security/#-oauth2-and-openid-connect) to see how to configure JHipster for Okta.
+### Use Okta
 
-**NEW:** You can use the [Okta CLI](https://github.com/oktadev/okta-cli) to add JHipster integration in seconds! After running `okta register`, run `okta apps create jhipster`. Then, source the created `.okta.env` file and start your app.
+See [JHipster's Okta Guide](https://www.jhipster.tech/security/#okta) to see how to configure JHipster for Okta. In addition to having a OIDC app for your JHipster backend, you'll need to [create a Native app on Okta](https://www.jhipster.tech/security/#create-a-native-okta-app-for-mobile) too.
```
Comment (mraible) [inline generators/ionic/resources/base/README.md:159]:
```suggestion
See [JHipster's Okta Guide](https://www.jhipster.tech/security/#okta) to see how to configure JHipster for Okta. In addition to having an OIDC app for your JHipster backend, you'll need to [create a Native app on Okta](https://www.jhipster.tech/security/#create-native-app-okta) too.
```

File: generators/ionic/resources/base/README.md [hunk 3]
```diff
@@ -145,172 +146,21 @@ If you see `java.io.IOException: Cleartext HTTP traffic to localhost not permitt
...
-    };
-    ```
-
-6. Restart your Ionic app and log in with Auth0!
+### Use Auth0
 
-    ```
-    npx cap run ios
-    ```
+See [JHipster's Auth0 Guide](https://www.jhipster.tech/security/#auth0) to see how to configure JHipster for Auth0. In addition to having a OIDC app for your JHipster backend, you'll need to [create a Native app on Auth0](https://www.jhipster.tech/security/#create-a-native-auth0-app-for-mobile) too.
```
Comment (mraible) [inline generators/ionic/resources/base/README.md:163]:
```suggestion
See [JHipster's Auth0 Guide](https://www.jhipster.tech/security/#auth0) to see how to configure JHipster for Auth0. In addition to having an OIDC app for your JHipster backend, you'll need to [create a Native app on Auth0](https://www.jhipster.tech/security/#create-native-app-auth0) too.
```

File: generators/ionic/resources/base/README.md [hunk 4]
```diff
@@ -12,11 +12,12 @@
...
 2. [Run on iOS](#run-on-ios)
 3. [Run on Android](#run-on-android)
-4. [Use Okta for Authentication](#use-okta-for-authentication)
-5. [Use Auth0 for Authentication](#use-auth0-for-authentication)
-6. [Services](#services)
-8. [i18n](#i18n) (adding languages)
-9. [Testing](#testing)
+4. [Use OpenID Connect for Authentication](#use-openid-connect-for-authentication)
+    - Configure [Okta](#use-okta)
+    - Configure [Auth0](#use-auth0)
```
Comment (mraible) [inline generators/ionic/resources/base/README.md:17]:
```suggestion
```

File: generators/ionic/resources/base/README.md [hunk 5]
```diff
@@ -12,11 +12,12 @@
 1. [Getting Started](#getting-started)
 2. [Run on iOS](#run-on-ios)
 3. [Run on Android](#run-on-android)
-4. [Use Okta for Authentication](#use-okta-for-authentication)
-5. [Use Auth0 for Authentication](#use-auth0-for-authentication)
-6. [Services](#services)
-8. [i18n](#i18n) (adding languages)
-9. [Testing](#testing)
+4. [Use OpenID Connect for Authentication](#use-openid-connect-for-authentication)
```
Comment (mraible) [inline generators/ionic/resources/base/README.md:15]:
```suggestion
4. [Use OpenID Connect for Authentication](#use-openid-connect-for-authentication) { Keycloak, Okta, and Auth0 }
```

**Comment by mraible** [comment]
@mshima Please merge at your convenience.

**Pull request merged**